ddsrouter_test/compose/test_cases/dds_rtps/ddsrouter.yaml

```diff
@@ -0,0 +1,21 @@
+version: v3.0
+
+xml:
+  raw: |
```

Comment:
Extend this test with more routers in different domains configured as follows:

- Loading the xml from the default file
- Loading the xml from the environment variable
- Loading the xml from a file

Reply:
Loading from default file requires a much more extensible change as the file will not be in the same workspade